test/itemrenderer.cpp

```diff
@@ -246,6 +246,75 @@ TEST_CASE("item_renderer::to_plain_text() renders text to the width specified "
 
 		REQUIRE(result == expected);
 	}
+
+
+	SECTION("angle brackets are only counted as one character") {
```

Comment:
This section is named the same as the next one. If one of them fails, the report will be a bit confusing. It'll contain line numbers, of course, so it *can* be figured out, but it'd be better to avoid the confusion in the first place.

The sections are pretty similar, so I'd remove the first one and keep the second one. Do you have a particular reason to keep the first one around?

Reply:
> Do you have a particular reason to keep the first one around?

No, just added it while debugging an issue.
Just removed it now.